### PR TITLE
doc: add license information (BSD-2-Clause)

### PR DESCRIPTION
**Pull request by Totktonada**

### CODE REVIEW

**Comment by Totktonada** [comment]
CI fails are due to https://github.com/moteus/luacov-coveralls/pull/30.

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Konstantin Nazarov <mail@knazarov.com>
+Andreev Semen <semen.andreev00@mail.ru>
+Alexey Kuzin <akudiyar@gmail.com>
+Alexander Turenko <alexander.turenko@tarantool.org>
+Alexey Romanov <romanov.alexey2000@gmail.com>
+Anastasia Neklepaeva <neknas28@gmail.com>
+Yaroslav Dynnikov <yaroslav.dynnikov@gmail.com>
+Vladimir Rogach <vladimir.rogach@gmail.com>
+Sergey Bronnikov <sergeyb@tarantool.org>
+Georgy Moiseev <georgy.moiseev@corp.mail.ru>
```
Comment (DifferentialOrange) [inline AUTHORS:16]:
```suggestion
Georgy Moiseev <moiseev.georgii@gmail.com>
```

**Review by DifferentialOrange — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (Totktonada) [reply AUTHORS]: Updated.

**Comment by Totktonada** [comment]
> CI fails are due to [moteus/luacov-coveralls#30](https://github.com/moteus/luacov-coveralls/pull/30).

Worked around in PR #270.

**Comment by Totktonada** [comment]
Rebased after PR #270.

**Pull request merged**